content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -269,13 +269,15 @@ Please see [Kubernetes issue #125689](https://github.com/kubernetes/kubernetes/i
 
 #### Deprecation of `status.nodeInfo.kubeProxyVersion` field for Nodes ([KEP 4004](https://github.com/kubernetes/enhancements/issues/4004))
 
-The `.status.nodeInfo.kubeProxyVersion` field of Nodes has been deprecated in Kubernetes v1.31,
-and will be removed in a later release.
+The `.status.nodeInfo.kubeProxyVersion` field of Nodes was deprecated in Kubernetes v1.29,
+and will not be populated in a later release.
 It's being deprecated because the value of this field wasn't (and isn't) accurate.
 This field is set by the kubelet, which does not have reliable information about the kube-proxy version or whether kube-proxy is running. 
 
 The `DisableNodeKubeProxyVersion` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will be set to `true` in by default in v1.31 and the kubelet will no longer attempt to set the `.status.kubeProxyVersion` field for its associated Node.
 
+Update: This was reverted to `false` in v1.31.1 to give a full year from deprecation announcement in v1.29, and is planned to be enabled by default in v1.33.
```

Comment:
I recommend:
- move this earlier
- use formatting (eg bold for Update, italic for the remaining text)